target/scripts/startup/setup.d/mail_state.sh

```diff
@@ -95,6 +95,11 @@ function _setup_save_states() {
 function _setup_adjust_state_permissions() {
   [[ ! -d ${DMS_STATE_DIR} ]] && return 0
 
+  # Parent directories must have executable bit set to descend the file tree for access,
+  # as each service running as a non-root user requires this to access their state directory,
+  # `/var/mail-state` must allow all users `+x`:
+  chmod o+x "${DMS_STATE_DIR}"
```

Comment:
Wondering if this should be `ugo+x`?

```suggestion
  chmod ugo+x "${DMS_STATE_DIR}"
```

Reply:
So ugo+x is just like `chmod +x` and I think should fine because the permissions inside /var/mail-state is still open than this 

```shell
drwxr-x--x  8 amavis  amavis  4096 Mar 18 04:36 lib-amavis
drwxr-xr-x  2 docker  docker  4096 Mar 18 04:56 lib-dovecot
drwxr-xr-x  2 root    root    4096 Dec 14  2022 lib-logrotate
drwxr-xr-x  2 postfix postfix 4096 Mar 17 22:09 lib-postfix
drwxr-xr-x 20 docker  docker  4096 Mar 17 22:00 qa.auto1.team
drwxr-xr-x 16 root    root    4096 Mar 18 04:56 spool-postfix
```

Reply:
I too think `+x` should be fine as it's the same as `ugo+x`.

Reply:
I didn't know that either 🤯